### PR TITLE
Added Kitchen tests and travis

### PR DESCRIPTION
**Pull request by Martin819**

### CODE REVIEW

**Comment by cloudlab-autobuild** [comment]
Can one of the admins verify this patch?

File: galera/master.sls [hunk 1]
```diff
@@ -128,17 +130,25 @@ mysql_bootstrap_update_maint_password:
...
+
+{%- endif %}
 
 galera_bootstrap_init_config:
   file.managed:
   - name: {{ master.config }}
   - source: salt://galera/files/my.cnf.init
   - mode: 644
   - template: jinja
+  {%- if not grains.get('noservices','false')%}
```
Comment (fpytloun) [inline galera/master.sls]: Should default to boolean False. The same bellow.

**Review by fpytloun — CHANGES_REQUESTED** [review]

**Comment by epcim** [comment]
@Martin819 poslal jsem ti PR na branch ktery to fixne. Jinak +1

File: galera/map.jinja [hunk 2]
```diff
@@ -1,7 +1,7 @@
 
 {%- set master = salt['grains.filter_by']({
   'Debian': {
-    'pkgs': ['mysql-server-wsrep-5.6', 'galera-3', 'rsync', 'python-mysqldb', 'psmisc', 'netcat', 'percona-xtrabackup', 'socat', 'libdbd-mysql', 'python-pymysql'],
+    'pkgs': ['mysql-wsrep-5.6', 'galera-3', 'rsync', 'python-mysqldb', 'psmisc', 'netcat', 'percona-xtrabackup', 'socat', 'libdbd-mysql', 'python-pymysql'],
```
Comment (epcim) [inline galera/map.jinja:4]: do I understand that current formula has wrong pkg name? mysql-server-wsrep ..  Pls explain.

Comment (Martin819) [reply galera/map.jinja:4]: Yes, exactly. The name has changed.

**Comment by Martin819** [comment]
@fpytloun can you plese recheck?

**Pull request merged**